x-pack/dockerlogbeat/docs/index.asciidoc

```diff
@@ -0,0 +1,21 @@
+= Elastic Log Driver for Docker
+
+:libbeat-dir: {docdir}/../../../libbeat/docs
+:log-driver: Elastic Log Driver
+:log-driver-long: Elastic Log Driver for Docker
+:log-driver-alias: elastic-logging-plugin
+:docker-version: ???
```

Comment:
@fearful-symmetry What is the minimum docker version that's required to use the plugin?

Reply:
Engine API 1.25, I believe.